sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -109,6 +112,16 @@ export interface ExponentialRetryPolicyOptions {
     retryDelayInMs?: number;
 }
 
+// @public
+export type FailoverHostGenerator = (retryState: RetryInformation, options: DefaultRetryPolicyOptions) => Generator<FailoverHostState | undefined, void, RetryInformation>;
```

Comment:
I think this should be an `Iterator` instead of a `Generator`, since there's not a real need to force the implementor to use a generator function to implement the iterator, it's just probably the most convenient way for them to do so.

Reply:
Done!